searchguard/searchguard.py

```diff
@@ -159,3 +159,22 @@ def view_role(role):
     else:
         # Raise exception because the role does not exist
         raise ViewRoleException('Error viewing the role {}, does not exist'.format(role))
+
+
+def view_user(user):
+    """Returns information about a Search Guard user (when the user exists)
+    The returned information contains the password hash and if present an array of the user's backend roles.
+    :param str user: Name of the user to view in Search Guard
+    """
+    if check_user_exists(user) is True:
```

Comment:
it is more pythonesque to just check for truthy instead of doing an identity level comparison with `is`

Reply:
also see https://github.com/ByteInternet/searchguard-python/pull/4